### PR TITLE
Imported data translations

### PR DESCRIPTION
**Pull request by tsubik**
Translations are provided in separate files with similar or the same structure as base language files. Additional data translations like: sectors, categories, metrics are in new `Data translations` file.

Some models became locale specific (`locale` column added) mostly because almost everything there is translated and introducing some fakes codes to be able to join base version and translation seemed like overkill.

For other data I'm adding jsonb column translations where we will hold all non-english translations. Simple module `Translate` has been introduced here to be able to indicate which attributes we want to translate. The way it works is similar to popular `globalize` or `mobile` gems so basically overriding attributes reader and writer to save and retrieve from the good column based on `I18n.locale`.

API endpoints support new parameter `locale` to get the data in other language (`id`)

The frontend part will be introduced in another PR.

As always

```
bundle exec rake db:migrate
bundle exec rake db:admin_boilerplate:create
bundle exec rake db:import
```

### CODE REVIEW

**Comment by simaob** [comment]
When running `rake db:migrate` I get the following error:

>>>NoMethodError: undefined method `translates' for Location (call 'Location.connection' to establish a connection):Class
Did you mean?  transaction


Am I missing something before that?

**Comment by tsubik** [comment]
Lemme check, I tested that on clean database from develop, but I see there are 2 location decorators, one is by mistake there, not sure which one.

**Comment by tsubik** [comment]
Ok, you can try now, try to run `spring stop` to make sure everything is fully reloaded.

File: app/models/concerns/translate.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module Translate
```
Comment (simaob) [inline app/models/concerns/translate.rb]: Nice module 👏

Comment (simaob) [reply app/models/concerns/translate.rb]: <3 metaprogramming

**Pull request merged**